site/config.toml

```diff
@@ -8,6 +8,7 @@ languageCode = "en-us"
 publishDir = "../docs"
 pygmentsCodeFences = true
 pygmentsStyle = "manni"
+enableEmoji = true
```

Comment:
@BenLloydPearson - When new documentation options are enabled via Hugo, the GitHub repo README file is updated with details on how to use the new feature. Love that we've added emoji support for developer docs pages -- please ensure that this functionality is described within the README for all contributors.